packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -137,14 +137,18 @@ const ProductList = (props) => {
     )
 
     /**************** Query Actions ****************/
+    // make sure that there is no invalid params sent to SCAPI
+    const searchParamsCopied = {...searchParams}
+    delete searchParamsCopied._refine
```

Comment:
You could probably do something a little slicker looking like below (I always thought `delete` was a little big of a less used pattern)

```
// eslint-disable-next-line @typescript-eslint/no-unused-vars
const {_refine, ...searchParamsCopy} =  searchParams
```